syz-fuzzer/fuzzer.go

```diff
@@ -616,6 +708,10 @@ func execute(pid int, env *ipc.Env, p *prog.Prog, needCover, minimized, candidat
 			continue
 		}
 		diff := cover.SignalDiff(maxSignal, inf.Signal)
+		if generatedByHint {
+			atomic.AddUint64(&statHintsWithNewSignal, 1)
```

Comment:
If we want to know what things resulted in new inputs, then we need to do this consistently for all things. I.e. how much inputs we from get generation/fuzzing/smashing/hints/etc.
This also need to be done in triageInput when we actually add it to corpus. Here it can be a duplicate or a coverage flake.
Since it gets larger I propose we do this in a separate followup commit.